pkg/eval/json_evaluator_model.go

```diff
@@ -40,6 +43,16 @@ func (f Flags) Merge(source string, ff Flags) (Flags, []StateChangeNotification)
 				FlagKey: k,
 			})
 		} else if !reflect.DeepEqual(val, v) {
+			if val.Source != source {
+				logger.Warn(
```

Comment:
You may want to update the viper doc, for `--uri`, it kinda conflicts with this now.

Reply:
updated viper doc